lib/grape/validations/validators/values.rb

```diff
@@ -17,12 +19,11 @@ def validate_param!(attr_name, params)
         excepts = @excepts.is_a?(Proc) ? @excepts.call : @excepts
         param_array = params[attr_name].nil? ? [nil] : Array.wrap(params[attr_name])
 
-        if !param_array.empty? && param_array.all? { |param| excepts.include?(param) }
-          raise Grape::Exceptions::Validation, params: [@scope.full_name(attr_name)], message: except_message
-        end
+        raise Grape::Exceptions::Validation, params: [@scope.full_name(attr_name)], message: except_message \
```

Comment:
You should make it a guard clause and this way you don't have to put it on multiple lines, so:

```ruby
return unless excepts.nil? || params_array.any? ...
```